File: Strings/__2000__/2150/2150.py
while True:
    try:
        alien_vowels = input()
        frase = input().split()
        count = 0
        
        for palavra in frase:
            for char in palavra:
                if char in alien_vowels:
                    count += 1
                    
        print(count)
    except EOFError:
        break

File: Strings/__2000__/2150/2150.ts
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");

while (input.length >= 2) {
    const alien_vowels = input.shift();
    const frase = input.shift().split(" ");
    let contador = 0;

    for (const palavra of frase) {
        for (const char of palavra) {
            if (alien_vowels.includes(char)) {
                contador++;
            }
        }
    }

    console.log(contador);
}
